
#*************** gocator_3200 node and device parameters ************************

#IP address of the camera device
ip_address: "10.208.2.17"
# ip_address: "10.8.1.10"

#run mode: run mode: 0->SNAPSHOT, 1->PUBLISHER, 2->SAVER
run_mode: 1

#loop rate in hz (good values: 10 when snapshot, 0.1 when publisher)
rate: 0.1

#frame name to which point cloud is referenced
frame_name: "gocator_point_cloud"

#camera exposure [us]
exposure: 10000

#camera point cloud grid spacing [mm]
spacing: 0.1
